wiki/en/misc/1-index.html

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jamulus ‒ Play music online. With friends. For free."
+title: "Jamulus — Play music online with friends."
 lang: "en"
 permalink: /
 layout: mainhomepage
 ldjAppCategory: "Music"
-ldjSlogan: "Play music online. With friends. For free."
+ldjSlogan: "Play music online with friends."
 ldjCreator: "Volker Fischer and contributors"
-metadescription: "Jamulus is open source software which enables musicians to perform music, rehearse or just jam, all in real time over the Internet."
+metadescription: "Copylefted libre software for musicians to perform, rehearse or just jam — all in real time over the Internet."
 mAltProgIcon: "Jamulus icon"
-mTSlogan: "Play music online. With friends. For free."
+mTSlogan: "Play music online. With friends."
```

Comment:
Please leave this out for now. We should decide on a slogan in with a larger number of contributors based on the outcome of a discussion.

Reply:
Is someone going to be upset that "For free" is removed?
I assume no.

Reply:
Jamulus competes with commercial offerings, and its price is a differentiator.

Reply:
OK, lets put that into the context of donations, or similar.
Its inherent freedom is the discerning quality.